pygmt/tests/test_grdimage.py

```diff
@@ -93,9 +93,32 @@ def test_grdimage_over_dateline(xrgrid):
 
 
 @check_figures_equal()
-def test_grdimage_central_longitude(grid, fig_ref, fig_test):
+@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("proj_type", ["H", "W"])
+def test_grdimage_central_meridians(grid, proj_type, lon0, fig_ref, fig_test):
     """
-    Test that plotting a grid centred at different longitudes/meridians work.
+    Test that plotting a grid with different central meridians (lon0) using
+    Hammer (H) and Mollweide (W) projection systems work.
     """
-    fig_ref.grdimage("@earth_relief_01d_g", projection="W120/15c", cmap="geo")
-    fig_test.grdimage(grid, projection="W120/15c", cmap="geo")
+    fig_ref.grdimage(
+        "@earth_relief_01d_g", projection=f"{proj_type}{lon0}/15c", cmap="geo"
+    )
+    fig_test.grdimage(grid, projection=f"{proj_type}{lon0}/15c", cmap="geo")
+
+
+@check_figures_equal()
+@pytest.mark.parametrize("lat0", [0, 30])
+@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("proj_type", ["Q", "S"])
+def test_grdimage_central_meridians_and_standard_parallels(
+    grid, proj_type, lon0, lat0, fig_ref, fig_test
+):
+    """
+    Test that plotting a grid with different central meridians (lon0) and
+    standard_parallels (lat0) using Cylindrical Equidistant (Q) and General
+    Stereographic (S) projection systems work.
+    """
+    fig_ref.grdimage(
+        "@earth_relief_01d_g", projection=f"{proj_type}{lon0}/{lat0}/15c", cmap="geo"
+    )
+    fig_test.grdimage(grid, projection=f"{proj_type}{lon0}/{lat0}/15c", cmap="geo")
```

Comment:
This check is meant to fix @MarkWieczorek's longstanding issue at #390 (we're getting there). Specifically, we check plotting xarray grids on different central meridians (lon0) at 0, 123 and 180deg, **and** different standard parallels (lat0) 0, 30deg for Cylindrical Equidistant (Q) and General Stereographic (S) projections systems. These plots are compared to a baseline/reference plot using a NetCDF grid directly.

**Good news**: With GMT 6.1.1, plotting these grids don't crash or give a black image anymore. Also for a few lon0/lat0 combinations e.g. (Q0/0, Q0/30, S123/0, S180/0), the plots are perfect.

**Bad news**: Certain lon0/lat0 combinations plot in xarray, but aren't exactly the same as the NetCDF plots. There's an RMS difference of ~25 for the Cylindrical Equidistant plots Q123/0, Q123/30, Q180/0, Q180/30 (see my comment at https://github.com/GenericMappingTools/pygmt/pull/560#issuecomment-687452674 for a visual diff). The General Stereographic plots also show minor RMS differences <2 for S0/0, S0/30, S123/30, S180/30, with the diff appearing as a strip along 0deg longitude (i.e. Greenwich).

I've also tested combinations of `GMT_IN|GMT_IS_REFERENCE` (the current setting), `GMT_IN`, and `GMT_IN|GMT_IS_DUPLICATE`, but the results are the same. Paul's comment at https://github.com/GenericMappingTools/gmt/pull/3829#issuecomment-668780568 does suggest that there might be outstanding issues to resolve here.

Reply:
Thanks for taking to the time to look into this. My attitude is that if you think you have done all you can with GMT 6.1.1, then you should release pygmt 0.2 as soon as possible, even if there are still some bugs that might be upstream. 

pygmt 0.1.2 is completely unusable for me (for plotting global projected data with xarray grids), and as far as I can tell with my tests from pygmt/master, things *appear* to be ok (i.e., no kernel crashes, and the images appear to be fine without detailed inspection). Nevertheless, the problem with the last column on the right of the plot being incorrect is a clear bug and will need to resolved in the following release.

Reply:
It seems there is nothing we can do in PyGMT to fix these bugs. We could do a 0.2.0 release soon.